### PR TITLE
feat: add wait to UtamElement

### PR DESCRIPTION
**Pull request by lizaiv77**
Add "wait" boolean property to element
Docs https://github.com/salesforce-experience-platform-emu/utam-docs/pull/840

### CODE REVIEW

File: utam-compiler/src/main/java/utam/compiler/grammar/JsonDeserializer.java [hunk 1]
```diff
@@ -133,7 +134,7 @@ public UtamPageObject deserialize(JsonParser parser, DeserializationContext ctxt
    *
    * @return the Page Object translation context
    */
-  public final TranslationContext getPageObjectContext() {
+  public TranslationContext getPageObjectContext() {
```
Comment (lizaiv77) [inline utam-compiler/src/main/java/utam/compiler/grammar/JsonDeserializer.java:137]: redundant on final class

Comment (jimevans) [reply utam-compiler/src/main/java/utam/compiler/grammar/JsonDeserializer.java:137]: Fair enough.

**Review by jimevans — APPROVED** [review]
LGTM. Looks like good test coverage.

**Review by jimevanssfdc — APPROVED** [review]
Accidentally reviewed logged into my other account.

**Pull request merged**